src/main/java/com/example/demo/Carddata/Carddata.java

```diff
@@ -0,0 +1,36 @@
+package com.example.demo.Carddata;
```

Comment:
Branches should not be named "review", derive branch name from task, for example "feature-https://github.com/KacFlor/Shop_Spring/pull/1-create-object-model"

Reply:
done